specs/jsonschema-core.md

```diff
@@ -2188,7 +2195,7 @@ and only allows the "data" and "children" properties. An example instance with
 
 ```jsonschema "Tree schema, extensible"
 {
-  "$schema": "https://json-schema.org/draft/next/schema",
+  "$schema": "https://json-schema.org/1/2025",
```

Comment:
Putting the iteration before the year is not going to sort very well, and I imagine it could easily be confused with a date.

Reply:
The compatibility guarantee stated [here](https://github.com/json-schema-org/json-schema-spec/pull/1596#discussion_r2046201670) necessitates that the version be first.